src/generators/csharp/CSharpGenerator.ts

```diff
@@ -52,25 +52,29 @@ export class CSharpGenerator extends AbstractGenerator<CSharpOptions, CSharpRend
     }
 
     const outputModel = await this.render(model, inputModel);
+
+    const outputDependencies = outputModel.dependencies.length === 0 ? '' : `${outputModel.dependencies.join('\n')}
+
+`;
```

Comment:
```suggestion
    const outputDependencies = outputModel.dependencies.length === 0 ? '' : `${outputModel.dependencies.join('\n')} \n \n`
```
Would this make it better for the eye? 🤔

Reply:
I suppose 🙂